aspnet/publishing/iis.rst

```diff
@@ -54,6 +54,27 @@ For more information on the ASP.NET Core Module, including configuration of the
 Application configuration
 -------------------------
 
+Enabling the `IISIntegration` service
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Ensure that the `UseIISIntegration()` is injected:
+
+.. code-block:: csharp
+
+  var host = new WebHostBuilder()
+    .UseKestrel()
+    .UseIISIntegration()
+    .UseStartup<Startup>()
+    .Build();
```

Comment:
Let's make this section mirror the `cli-samples`, so make it include `.UseContentRoot(...)` ...
```c#
var host = new WebHostBuilder()
    .UseKestrel()
    .UseContentRoot(Directory.GetCurrentDirectory())
    .UseIISIntegration()
    .UseStartup<Startup>()
    .Build();
```